lib/jxl/splines.cc

```diff
@@ -272,12 +272,13 @@ Vector operator-(const Spline::Point& a, const Spline::Point& b) {
   return {a.x - b.x, a.y - b.y};
 }
 
-std::vector<Spline::Point> DrawCentripetalCatmullRomSpline(
-    std::vector<Spline::Point> points) {
-  if (points.size() <= 1) return points;
+// TODO(eustas): avoid making a copy of "points".
+void DrawCentripetalCatmullRomSpline(std::vector<Spline::Point> points,
+                                     std::vector<Spline::Point>& result) {
+  // TODO(eustas): in spec "point" splines are allowed
+  if (points.size() <= 1) return;
```

Comment:
Why don't we push_back() points[0] in result?

Reply:
Exactly! That is why I have left the comment.

Reply:
Fixed. PTAL